schemas/PollCreated.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "object",
+  "description": "Triggered when a poll is created.",
+  "properties": {
+    "eventName": {
+      "enum": ["PollStarted"]
+    },
+    "isAnonymous": {
```

Comment:
This is the disclosed/undisclosed things? Votes are never anonymous. 🤔

Reply:
In the matrix spec it is called "disclosed"/"undisclosed". In the UI it is called "anonymous" true/false. Not sure what nomenclature the analytics usually follow.

Reply:
I think @alfogrillo wants to point out that the votes can be seen, so even if the UI does not tell who has chosen which option, it's not hard to see that information by inspecting content of (hidden) events in the timeline.

Reply:
This is well known within the team.
Design/Product decided to use `anonymous`  in the user visible copy though.
I don't know if analytics tend to follow the product wording or than the Matrix Spec wording. Do you have any guidance on this?

Reply:
We should be consistent in deciding which naming convention we want to use. 
To me using the **spec one** is more robust, since copies in the UI can change much more easily.